Common/src/main/resources/assets/hexcasting/lang/en_us.flatten.json5

```
@@ -890,6 +890,11 @@
         cos: "the cosine of %s",
       },
 
+      invalid_operator_args: {
+        single: "Got an unexpected iota at index %d of the stack: %s",
+        plural: "Got %d unexpected iotas at indices %d-%d of the stack: %s",
```

nit: the leading case is inconsistent across mishaps, but `invalid_value` is lowercase, so I think it makes sense to match that

```suggestion
        single: "got an unexpected iota at index %d of the stack: %s",
        plural: "got %d unexpected iotas at indices %d-%d of the stack: %s",
```